packages/datadog-instrumentations/src/cucumber.js

```diff
@@ -0,0 +1,82 @@
+'use strict'
+
+const { addHook, channel } = require('./helpers/instrument')
+const shimmer = require('../../datadog-shimmer')
+
+const runStartCh = channel('ci:cucumber:run:start')
+const runEndCh = channel('ci:cucumber:run:end')
+const runAsyncEndCh = channel('ci:cucumber:runAsync:end')
+const runStepStartCh = channel('ci:cucumber:runStep:start')
+const runStepEndCh = channel('ci:cucumber:runStep:end')
+const errorCh = channel('ci:cucumber:error')
+
+function wrapRun (pl, isLatestVersion) {
+  shimmer.wrap(pl.prototype, 'run', run => function () {
+    if (!runStartCh.hasSubscribers) {
+      return run.apply(this, arguments)
+    }
+
+    runStartCh.publish({ pickleName: this.pickle.name, pickleUri: this.pickle.uri })
```

Comment:
not to self: I just realized: the source root needs to be stripped off from `this.pickle.uri`